documentation/source/QuickstartBasicToolkit.md

```diff
@@ -0,0 +1,245 @@
+# The Basic Skills of SG
+
+Learn the basics of model development with SuperGradients. Researchers and machine learning engineers should start here.
```

Comment:
please change:
In this tutorial we will go over all of the besic functionalities of SuperGradients very briefly. 
Go over the next sections to quickly learn how to train, test and predict using SuperGradients. 
Check-out our extended tutorials on the verious features you can find in SuperGradients, and task specific guids.

Reply:
Fixed https://github.com/Deci-AI/super-gradients/pull/718/commits/d9199dfa4e657910a027f3acfd768c85b6a8fa28